api/config/config_local.json

```diff
@@ -93,7 +93,9 @@
   "absorb": {
     "externalDepartmentId": "Development",
     "externalDepartmentIdPopulatedForNewUsers": true,
-    "enabledForNewUsers": false,
+    "enabledForNewUsers": true,
+    "rtTrainingCourseId": "9ad49c70-3b72-4789-8282-5794efcd4ce1",
```

Comment:
Is this information important enough to be inside a file in a bucket as we have an open code base

Reply:
I don't think this needs to be hidden. In order to do anything with the ID, you would need the right Absorb permissions. The only ways to access the course are through a user account or using an API key. The API key is secure and is kept in a bucket.

This value is also present in the course URL, so all users that take the course would see it anyway.